lib/mix/tasks/dogma.ex

```diff
@@ -11,8 +11,11 @@ defmodule Mix.Tasks.Dogma do
     {dir, formatter} = argv |> parse_args
     config = Config.build
 
+    {:ok, events} = GenEvent.start_link([])
```

Comment:
I feel Dogma should probably crash if the reporters fail.

Again, I've not really got into OTP much, so let me ask this. Due to the use of `start_link/2` if the event handler process fails the parent process will also fail, as they are connected together, correct?
If that's the case, this implementation means they both crash, which I think is what we want.